tests/eko/scale_variations/test_exponentiated.py

```diff
@@ -21,11 +21,11 @@ def test_gamma_singlet_fact():
     gamma_s = np.array([1.0, 0.5, 0.25, 0.125])
     gamma_s_LO_0 = gamma_variation(gamma_s.copy(), (1, 0), 3, 0)
     np.testing.assert_allclose(gamma_s_LO_0, gamma_s)
-    gamma_s_LO_1 = gamma_variation(gamma_s.copy(), (1, 0), 3, 1)
+    gamma_s_LO_1 = gamma_variation(gamma_s.copy(), (1, 0), 3, -1)
     np.testing.assert_allclose(gamma_s_LO_1, gamma_s)
-    gamma_s_NLO_1 = gamma_variation(gamma_s.copy(), (2, 0), 3, 1)
+    gamma_s_NLO_1 = gamma_variation(gamma_s.copy(), (2, 0), 3, -1)
     assert gamma_s_NLO_1[1] < gamma_s[1]
-    gamma_s_NNLO_1 = gamma_variation(gamma_s.copy(), (3, 0), 3, 1)
+    gamma_s_NNLO_1 = gamma_variation(gamma_s.copy(), (3, 0), 3, -1)
```

Comment:
Can you remind me the reason of the final `-1`?

Reply:
in this PR $\xi_F$ was flipped on his head, i.e. $L\to -L$, and the quickest way to fix the benchmark was to undo this change

Reply:
Ok I see, I believed you flipped the sign and reverted to the original

Reply:
the unit test is old so it follows the old convention